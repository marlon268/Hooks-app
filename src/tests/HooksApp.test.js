import React from "react";
import { shallow } from "enzyme";
import { HooksApp } from "../HooksApp";

describe("Pruebas al componente <HooksApp/>", () => {
   test("Debe de mostrar el componente <HooksApp/>", () => {
      const wrapper = shallow(<HooksApp />);
      expect(wrapper).toMatchSnapshot();
   });
});
