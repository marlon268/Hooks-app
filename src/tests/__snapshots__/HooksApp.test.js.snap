// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente <HooksApp/> Debe de mostrar el componente <HooksApp/> 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
